src/main/resources/templates/email/TechnicalQuestions_Interviewee.txt

```diff
@@ -0,0 +1,33 @@
+Hello {{interviewee_first_name}},
+
+We see that you registered for a {{position}} mock interview with InterviewMe on {{formatted_date}}. As these interviews usually come with technical questions, we wanted to share with you some tips.
+
+  Repeat/clarify the question with the interviewer.
+    - Pay attention for hints.
+    - Gives you more time to digest the question.
+    - Vagueness is expected. This gives you the chance to define the problem better.
+
+  Brainstorm potential solutions.
+    - Start with the obvious/worst case.
+    - Draw pictures.
+    - Evaluate potential solutions by running time.
+    - Remember, this is a conversation, not a monologue.
+
+  Before coding:
+    - Make sure you understand your approach.
+    - Make sure the interviewer understands your approach.
+    - You may want to pick a suboptimal approach if it's easier to code (although often the optimal approach is the simplest.)
+    - Think about APIs, function signatures, etc.
+
+  During coding:
+    - Small syntax/language issues are OK, but avoid pseudo-code.
+    - Subroutines can help clarify code/allow you to skip writing some code.
+    - Style counts; use semantically meaningful names.
+
+  After coding:
+    - Immediately work through a small example to try and catch any logic/errors.
```

Comment:
The coming up with an example should be during the before coding conversation and used throughout the process to check their code as they go.

Reply:
l pulled this from Jack's lnterview Tips, which was pulled from some Google resource. l think it makes sense to work through an example at the end to simulate testing it.

Reply:
I've found this is a personal preference that differs between interviewees. Having an example to work with as they work through the problem can be helpful for some candidates but would be a distraction for others (generally skilled candidates who are able to formulate a solution quickly). This also depends a great deal on the question, and for certain questions (like the one I will probably ask you as a mock interview), examples can actually be misguiding.

Reply:
Will remove this line.